console/src/components/Services/Data/TableBrowseRows/ViewActions.js

```diff
@@ -73,10 +73,14 @@ const vMakeRowsRequest = () => {
     const requestBody = {
       type: 'bulk',
       args: [
-        generateSelectQuery(
+        getSelectQuery(
```

Comment:
I am keeping this change because:
1. One of the reasons behind using util functions is to not make them assume a structure
2. I don't like a new object being created and destroyed everytime this function is being called, when we can totally get away with using arguments.
3. I don't think this is easier or harder because editor can help with knowing arguments. It is equally easy or hard to use objects or function arguments without editor support.

It made sense in case of `runQuery` as it added to the readability because `runQuery` has a context of its own and it's convenient to explicitly see what is being overridden when it is being overriden. No such reason for `getSelectQuery`.